CHANGELOG.md

```diff
@@ -23,6 +23,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
    methods no longer ignore `Process.IOPriority` and `Process.Scheduler`
    settings. (#4585)
 
+### Added
+ * CPU affinity support for `runc exec`. (#4327)
```

Comment:
Late to the party, but https://github.com/opencontainers/runc/blob/main/docs/spec-conformance.md had to be synced with the spec v1.2.1

Reply:
Wanna make a PR so it's included in the next rc? :)

Reply:
#4660